modules/network-observability-flowcollector-api-specifications.adoc

```diff
@@ -1107,72 +1137,167 @@ Type::
 |===
 | Property | Type | Description
 
-| `authToken`
-| `string`
-| `authToken` describes the way to get a token to authenticate to Loki. +
- - `DISABLED` does not send any token with the request. +
- - `FORWARD` forwards the user token for authorization. +
- - `HOST` [deprecated (*)] - uses the local pod service account to authenticate to Loki. +
- When using the Loki Operator, this must be set to `FORWARD`.
+| `advanced`
+| `object`
+| `advanced` allows setting some aspects of the internal configuration of the Loki clients. This section is aimed mostly for debugging and fine-grained performance optimizations.
 
-| `batchSize`
-| `integer`
-| `batchSize` is the maximum batch size (in bytes) of logs to accumulate before sending.
+| `enable`
+| `boolean`
+| Set `enable` to `true` to store flows in Loki. It is required for the {product-title} Console plugin installation.
+
+| `lokiStack`
+| `object`
+| Loki configuration for "LokiStack" mode. This is useful for an easy loki-operator configuration. It is ignored for other modes.
+
+| `manual`
+| `object`
+| Loki configuration for "Manual" mode. This is the most flexible configuration. It is ignored for other modes.
 
-| `batchWait`
+| `microservices`
+| `object`
+| Loki configuration for "Microservices" mode. Use this option when Loki is installed using the microservices deployment mode (https://grafana.com/docs/loki/latest/fundamentals/architecture/deployment-modes/#microservices-mode). It is ignored for other modes.
+
+| `mode`
 | `string`
-| `batchWait` is the maximum time to wait before sending a batch.
+| `mode` must be set according to the installation mode of Loki: +
+ - Use "LokiStack" when Loki is managed using the Loki Operator +
+ - Use "Monolithic" when Loki is installed as a monolithic workload +
+ - Use "Microservices" when Loki is installed as microservices, but without Loki Operator +
+ - Use "Manual" if none of the options above match your setup +
```

Comment:
@jotak I think we should put these `mode` values in ` ` instead of " "

Reply:
hmm I thought we were using " " when we refer to string yaml values, as opposed to ` ` for keys / paths and perhaps a few other things ...
I need to check if we do that in other places to be consistent

Reply:
ok, you're right :-) I'll change upstream